src/main/java/com/adyen/model/checkout/PaymentMethod.java

```diff
@@ -87,12 +87,7 @@ public PaymentMethod configuration(Map<String, String> configuration) {
     }
 
     public PaymentMethod putConfigurationItem(String key, String configurationItem) {
-
-        if (this.configuration == null) {
-            this.configuration = null;
```

Comment:
I think this line was wrong before and should be fixed, not removed (same case as in InputDetail)

Reply:
Fixed.